virocon/distributions.py

```diff
@@ -249,7 +461,32 @@ def _fit_lsq(self, data):
 
         
 class LogNormalDistribution(Distribution):
+    """
+    A Lognormal Distribution. 
     
+    :math:`f(x) = \\frac{1}{x\\widetilde{\\sigma} \\sqrt{2\\pi}}\\exp \\left[ \\frac{-(\\ln x - \\widetilde{\\mu})^2}{2\\widetilde{\\sigma}^2}\\right]`
+     
+    
+    Parameters
+    ----------
+    mu : float
+        Mean parameter of the lognormal distribution. Defaults to 0.
+    sigma : float
+        Variance parameter of the lognormal distribution. Defaults to 1.
+    f_mu : float
+        Fixed mean parameter of the lognormal distribution (e.g. given physical
+        parameter). If this parameter is set, mu is ignored. Defaults to 
+        None.
+    f_sigma : float
+       Fixed variance parameter of the lognormal distribution (e.g. given 
+       physical parameter). If this parameter is set, sigma is ignored. 
+       Defaults to None. 
+    fit_method : float
+        Method of estimating the parameters of a probability distribution. 
+        Defaults to maximum likelihood estimation (mle).
+    weights : KAI???
```

Comment:
Weights for weighted-least-squares fitting. Ignored (not implemented) for LogNormalDistribution.